tests/run_standalone_tests.sh

```diff
@@ -26,18 +26,17 @@ export PL_RUN_STANDALONE_TESTS=1
 defaults=" -m coverage run --source ${source} --append -m pytest --no-header -v -s --timeout 120 "
 echo "Using defaults: ${defaults}"
 
-# get the testing location as the fist argument
+# get the testing location as the first argument
 test_path=$1
 printf "source path: $test_path\n"
 
 # collect all tests with parametrization based filtering with PL_RUN_STANDALONE_TESTS
 standalone_tests=$(python3 -m pytest $test_path -q --collect-only --pythonwarnings ignore)
-printf "Collected tests: \n $standalone_tests"
+printf "Collected tests: \n $standalone_tests\n"
 # match only lines with tests
-parametrizations=$(grep -oP '\S+::test_\S+' <<< "$standalone_tests")
+parametrizations=$(perl -nle'print $& while m{\S+::test_\S+}g' <<< "$standalone_tests")
```

Comment:
This looks odd, shouldn't there be a space here between argument and value?
```suggestion
parametrizations=$(perl -nle 'print $& while m{\S+::test_\S+}g' <<< "$standalone_tests")
```

Reply:
Good catch, thanks. I did the same below for the other call.